HH_custom_primary_key.md

```diff
@@ -0,0 +1,97 @@
+###Example: Create a model with custom primary key `name:string` instead of autogenerated `id:integer`
+
+IMPORTANT NOTE: this example was chosen for simplicity.  It is not advisable to use something that is likely to change
```

Comment:
Please make text soft-wrap so that it flows well in readme.io. They will insert their own line breaks, and the text will become really choppy otherwise.